LEGO1/mxobjectfactory.cpp

```diff
@@ -28,16 +28,14 @@ MxObjectFactory::MxObjectFactory()
 // OFFSET: LEGO1 0x100b12c0
 MxCore *MxObjectFactory::Create(const char *p_name)
 {
-  MxCore* object = NULL;
```

Comment:
It looks like there is a a major accuracy regression with this one

Reply:
I reverted the change. It was not needed for mingw compatibility.